examples/mesh-shading/data/index.html

```diff
@@ -0,0 +1,17 @@
+<html>
+  <head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type"/>
```

Comment:
why do you need an HTML here? is this targeting JS?

Reply:
It is a leftover. As you correctly guessed the mesh example is based on the quad example. It is basically just a copy paste with removed image and few modifications here and there.

I would like to achieve support of mesh shaders in wgpu(-rs) so I left the html there as It could possibly run in the Firefox then. But that might be too much in a future so I am removing It.